src/Controls/tests/BindingSourceGen.UnitTests/DiagnosticsTests.cs

```diff
@@ -4,7 +4,7 @@ namespace BindingSourceGen.UnitTests;
 
 public class DiagnosticsTests
 {
-    [Fact]
+    [Fact(Skip = "Improve detecting overloads")]
```

Comment:
What's wrong with the test? Does it report wrong diagnostic?

Reply:
As we based our overload detection on checking the type of the arguments now it will report that we are using a wrong overload as the second argument is not lambda.

Reply:
OK. It's acceptable that it reports BSG0004 for now. Let's deal with this later and merge it as is.